lib/misc.js

```diff
@@ -65,35 +65,51 @@ exports.generateId = function (fields) {
   return base32hex.encodeBuffer(hasher.digest(), { paddingChar: '-' });
 };
 
-exports.encryptArrayToFile = function (dataArray, filePath, env, serverSecret) {
-  var iv = `${env}-environment`.substring(0, 16);
-  var key = serverSecret.substring(0, 32);
-  var algorithm = 'aes-256-cbc';
+var userIdsConfig = function (env, serverSecret) {
+  const key = crypto.createHash('sha256').update(serverSecret).digest('hex');
```

Comment:
Rather than taking the digest as a hex string, just take the raw Buffer (which will have exactly 32 bytes), and use that for the key directly. So:

```suggestion
  const key = crypto.createHash('sha256').update(serverSecret).digest();
```
Then, below, you won't need `key: key.substring(0, 32)` and should just be able to use `key: key`.